DTS-STN/dts-rmp
# updated provinces names

# Description

Updated province names on contact form 

## Test Instructions

1. Nvigate to Add a contact you should see the list of provinces on the Prov / Territories 

## Help Requested

## Checklist
